backend/src/ai_service/base.py

```diff
@@ -0,0 +1,64 @@
+import abc
+from typing import List, Dict
+
+
```

Comment:
Requested task - MessagePassingInterface seems missing its implementation in this pull request